skimage/measure/_regionprops.py

```diff
@@ -819,6 +824,22 @@ def regionprops(label_image, intensity_image=None, cache=True):
     if not np.issubdtype(label_image.dtype, np.integer):
         raise TypeError('Label image must be of integer type.')
 
+    if coordinates is not None:
+        if coordinates == 'rc':
+            msg = ('The coordinates keyword argument to skimage.measure.'
+                   'regionprops is deprecated. All features are now computed '
+                   'in rc (row-column) coordinates. Please remove '
+                   '`coordinates="rc"` from all calls to regionprops before '
+                   'updating scikit-image.')
+            warn(msg, stacklevel=2)
+        else:
+            msg = ('Values other than "rc" for the "coordinates" argument '
+                   'to skimage.measure.regionprops are no longer supported. '
+                   'You should update your code to use "rc" coordinates and '
+                   'stop using the "coordinates" argument, or use skimage '
+                   'version 0.15.x or earlier.')
```

Comment:
```suggestion
                   'version 0.15.x or earlier. The functions"transpose" or "moveaxis"'
                   'may help you transition your code.')
```

Reply:
`np.transpose` and `np.moveaxis` ?

Reply:
Honestly I'm not super in-love with this suggestion. It seems to me to be too vague to be useful. We could add an "xy to rc transition guide", but I don't think it should hold up this PR or release...

Reply:
I agree it shouldn't hold up this PR, but the most frustrating thing to hit is:

> Aw man my code isn't working, let me type `pip install scikit-image=0.14` 

and users now get stuck on 0.14 for much longer than they should. Pointhing them to `np.moveaxis` or `np.transpose` alleviates that.

Reply:
or my favorite

`sudo pip install scikit-image=0.14`

which will totally break their modern systems in 1 year.

Reply:
My feeling is that you have to know exactly what you need to properly transition your code. Calling `np.transpose` on your image *may* fix your xy/rc problem, but it may not. For example, the orientation will change sign compared to the previous convention.